apps/explorer/src/const.ts

```diff
@@ -131,9 +131,9 @@ export const XDAI: TokenErc20 = {
   address: NATIVE_TOKEN_ADDRESS,
 }
 
-export const NATIVE_TOKEN_PER_NETWORK: Record<string, TokenErc20> = {
+export const NATIVE_TOKEN_PER_NETWORK: Record<SupportedChainId, TokenErc20> = {
   '1': ETH,
```

Comment:
Nitpick, to be consistent it's better to use `[SupportedChainId.MAINNET]` instead of `'1'`